### PR TITLE
update doc about vagrant

### PR DESCRIPTION
**Pull request by yaolingling**

### CODE REVIEW

File: docs/tutorials/vagrant.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ prerequisites
 
     which can also be used with a pager, such as `less`.
 
-You will need to have `Vagrant`_ and `VirtualBox`_ installed on your machine to use
+You will need to have `Vagrant`_ and `VirtualBox`_ (>=1.8.0) installed on your machine to use
```
Comment (anhou) [inline docs/tutorials/vagrant.rst]: @yaolingling vagrant 1.8.0 has problems, so it's better >=1.8.1 here  https://github.com/RackHD/RackHD/blob/master/example/README.md 
  1.8.0/1.8.1 is the version of vagrant, the virtualbox version is 4.3.x or 5.x.x

File: docs/tutorials/vagrant.rst [hunk 2]
```diff
@@ -79,18 +79,14 @@ The Vagrant setup also enables port forwarding that allows your localhost to acc
     git clone https://github.com/rackhd/rackhd
     cd rackhd/example
 
+- Edit the Vagrantfile - delete the line 89: 'v.gui = true'
```
Comment (anhou) [inline docs/tutorials/vagrant.rst]: why delete this line?

File: docs/tutorials/vagrant.rst [hunk 3]
```diff
@@ -767,15 +763,35 @@ Power On
...
 
 Install OS
 ^^^^^^^^^^^
+Three steps to install os.
+
+- step 1: OBM setting
+.. code-block:: REST
+    curl localhost:9090/api/current/nodes/<nodeId>/catalogs/bmc | jq '.' | grep "IP Address" 
+    curl -k -X PUT \
+    -H 'Content-Type: application/json' -H 'Authorization: JWT <token>' \
```
Comment (anhou) [inline docs/tutorials/vagrant.rst]: is there guidance how to get the token?

File: docs/tutorials/vagrant.rst [hunk 4]
```diff
@@ -767,15 +763,35 @@ Power On
...
-        https://localhost:9093/api/2.0/nodes/5799151faa0559c007dab5e3/workflows
+        https://localhost:9093/api/2.0/nodes/<nodeId>/workflows
 
 Install OS
 ^^^^^^^^^^^
+Three steps to install os.
+
+- step 1: OBM setting
+.. code-block:: REST
+    curl localhost:9090/api/current/nodes/<nodeId>/catalogs/bmc | jq '.' | grep "IP Address"
```
Comment (anhou) [inline docs/tutorials/vagrant.rst]: is this specific to Infrasim vnode, as I know, some vendors catalog's 'IP Address' don't have BMC IP

File: docs/tutorials/vagrant.rst (hunk 2 above)
Comment (yaolingling) [inline docs/tutorials/vagrant.rst]: If not, the quanta_d51 cannot set up successfully. The error message is about vm.gui.

File: docs/tutorials/vagrant.rst (hunk 3 above)
Comment (yaolingling) [inline docs/tutorials/vagrant.rst]: The guidance has been written in this doc.

File: docs/tutorials/vagrant.rst (hunk 1 above)
Comment (yaolingling) [inline docs/tutorials/vagrant.rst]: I will change the version later. Thanks.

File: docs/tutorials/vagrant.rst (hunk 4 above)
Comment (yaolingling) [inline docs/tutorials/vagrant.rst]: Vagrant is used to create vnode.  So, the catalog contains 'IP Address'.

File: docs/tutorials/vagrant.rst (hunk 2 above)
Comment (anhou) [inline docs/tutorials/vagrant.rst]: if v.gui=true, have problems, I suggest fix there directly

**Pull request closed without merging**